kong/plugins/basic-auth/handler.lua

```diff
@@ -1,20 +1,24 @@
 -- Copyright (C) Kong Inc.
-
 local BasePlugin = require "kong.plugins.base_plugin"
 local access = require "kong.plugins.basic-auth.access"
 
+
 local BasicAuthHandler = BasePlugin:extend()
 
+
 function BasicAuthHandler:new()
   BasicAuthHandler.super.new(self, "basic-auth")
 end
 
+
 function BasicAuthHandler:access(conf)
   BasicAuthHandler.super.access(self)
   access.execute(conf)
 end
 
+
 BasicAuthHandler.PRIORITY = 1001
-BasicAuthHandler.VERSION = "0.1.0"
+BasicAuthHandler.VERSION = "0.1.1"
```

Comment:
oh, my PR did not upgrade the VERSION identifier of any plugins. Should we do that?

Reply:
Probably doesn't matter really. I updated the version as it is different than the previous. Only the minor though.